apps/site/assets/ts/schedule/components/LineDiagram.tsx

```diff
@@ -70,7 +80,17 @@ const LineDiagram = ({
               <div className="m-schedule-line-diagram__connections">
                 {filteredConnections(routeStop.connections).map(
                   (route: Route) => (
-                    <a href={`/schedules/${route.id}/line`} key={route.id}>
+                    <a
+                      href={`/schedules/${route.id}/line`}
+                      key={route.id}
+                      data-toggle="tooltip"
+                      data-trigger="focus"
+                      data-placement="bottom"
+                      data-animation="false"
+                      data-selector="true"
+                      data-original-title={connectionName(route)}
```

Comment:
I'm curious, why do we need both `title` and `data-original-title`?

Reply:
This reflects a total Bootstrap v.4.0.0-alpha.2 quirk: adding the `data-selector="true"` and `data-original-title` intercepts the tooltip's normal behavior of resetting the `title` to "" and inserting the title into `data-original-title` upon activation. I'd like to preserve the title rather than ever setting it to an empty string, for accessibility's sake.

Reply:
Ah, that does seem important. Do we know which version of Bootstrap fixes this? Can we call it out in a comment somewhere so we remember to remove it when the time comes?